src/utils/private.ts

```diff
@@ -135,7 +133,7 @@ export const getWideConfig = async (context: Context) => {
 
   const parsedOrg: WideOrgConfig | undefined = parseYAML(orgConfig);
   const parsedRepo: WideRepoConfig | undefined = parseYAML(repoConfig);
-  const parsedDefault: WideRepoConfig | undefined = getDefaultConfig();
+  const parsedDefault: WideRepoConfig = getDefaultConfig()!;
```

Comment:
You sure about this `!` ?

Reply:
We need to tell Typescript that we know this will be parsed correctly because if we don't put `!` Typescript will complain that the value might be undefined and we have to handle that. 

We can only remove `!` if we use another value as default if the `defaultConfig` is undefined

Reply:
Then don't return `undefined` in :86. You're not using types correctly if that is your intent.